_blog/2024-02-09-using-clerk-in-rust.mdx

```diff
@@ -0,0 +1,1014 @@
+---
+title: "Using Clerk authentication in Rust"
+description: This article is on how to use Clerk authentication in Rust based back-end applications and little bit of an extension on integration of shared database using Shuttle. 
+author: sourabpramanik
+tags: [rust, clerk, vite, guide]
+thumb: 
+cover: 
+date: 
+---
+
+In this article, I have shared how I built an Issue tracking application using Vite for the front-end, and [Shuttle](https://www.shuttle.rs/) and [Actix web](https://actix.rs/) for the back-end. In addition to that I have also used [Clerk](https://clerk.com/) for authentication in the front-end and protecting the endpoints in the back-end.
+
+Here is the [source code](https://github.com/sourabpramanik/issue-tracker) of the complete project if you want to follow along.
+
+To start this project I have used the template from Shuttle’s [example repository](https://github.com/shuttle-hq/shuttle-examples) that has a barebone setup of Vite, Shuttle, Actix web, and Clerk. 
```

Comment:
Can you also add the link to the project running on Shuttle?

Reply:
Sorry I missed the main thing to mention my bad!